pulser-simulation/pulser_simulation/qutip_result.py

```diff
@@ -126,33 +126,64 @@ def _weights(self) -> np.ndarray:
         # Takes care of numerical artefacts in case sum(weights) != 1
         return cast(np.ndarray, weights / sum(weights))
 
+    def _get_relevant_subsystem_indices(
+        self, to_basis: str, tol: float
+    ) -> List[int]:
```

Comment:
We actually don't need `List` here, we can (and should since python 3.9 was released) use `list` instead